compiler/tests-spec/testData/diagnostics/linked/statements/assignments/p-2/neg/1.2.fir.kt

```diff
@@ -8,7 +8,7 @@
  */
 fun case1() {
     val x : Case1? = Case1()
-    x.<!INAPPLICABLE_CANDIDATE!>x<!> = "0"
+    x.x = "0"
```

Comment:
Note that this test intentionally adds `-UNSAFE_CALL` to focus on `val` reassignments.